src/main/java/edu/kis/vh/nursery/FIFORhymer.java

```diff
@@ -1,18 +1,34 @@
 package edu.kis.vh.nursery;
 
+/**
+ * @author Paulina Stelmasiak
+ */
+
 public class FIFORhymer extends DefaultCountingOutRhymer {
 
-	public DefaultCountingOutRhymer temp = new DefaultCountingOutRhymer();
+	private final DefaultCountingOutRhymer temp = new DefaultCountingOutRhymer();
+
+    /**
+     * Metoda DefautlCountingOutRhymer
+     * @return temp
+     */
+	public DefaultCountingOutRhymer getTemp() {

```

Comment:
Shouldn't be encapsulated (typical private field). This is an exaggeration. Using private fields internally usually do not require encapsulation.